00 Meta/Vault Infos/Introduce Use Cases.md

```diff
@@ -0,0 +1,44 @@
+# Introduce Use Cases
+
+In order to make the vault better navigable and give some more context to available queries as well as inspiration how these queries could be used, we want to introduce **Use Cases**. 
+
+## What's a good topic for a Use Case?
+
+Like the name says, a Use Case should be about some specific scenario you could achieve with dataview which requires (or can require) more than one dataview query. They should be **overviews for a topic**. Some examples:
+- A Daily/Weekly/Quarterly Note (Template)
+- A Music Album Overview page that provides information for songs, the album, the interpret, ...
+- A persons page with information about his birthday, the last contact with them, their interests, their relationships with other persons, ...
+- Overview about a TV series, it's prequels and sequels, how many episodes you watched, when the new series comes out, participating actors, director, world, ... 
+
+## Parts of a Use Case
+
+A **Use Case** should consist of the following:
```

Comment:
I think that if you write "consist of the following", it makes more sense to use an enumeration, in stead of headers? What do you think?